eos/saveddata/fit.py

```diff
@@ -461,11 +461,16 @@ def calculateModifiedAttributes(self, targetFit=None, withBoosters=False, dirtyS
         timer = Timer(u'Fit: {}, {}'.format(self.ID, self.name), logger)
         logger.debug("Starting fit calculation on: %r, withBoosters: %s", self, withBoosters)
 
+        try:
+            projectionInfo = self.getProjectionInfo(targetFit.ID)
+        except:
+            pass
+        else:
+            logger.debug("ProjectionInfo: %s", projectionInfo)
+
```

Comment:
@blitzmann Hmm, for some reason these are hidden and say "outdated".  Not sure why.  And since you didn't ping me on it, I didn't get a notification that it had been updated.  vOv

Anyway.  As I said previously

>Also, I did want you to look over one commit in particular:
>4036ac8
>
>This one I started working on, with the goal of moving projectionInfo out of if targetFit: so that pyCharm understands that it is indeed created before it's referenced, but you can see where I stopped.
>
>It works fine as is, but I feel like we should handle it a bit better.